### PR TITLE
Do not include starknet accounts file in commits

### PR DESCRIPTION
**Pull request by piwonskp**
Changed its location to `tests/cli/` since the file is in `.gitignore`

### CODE REVIEW

File: tests/behaviour/behaviour.test.ts [hunk 1]
```diff
@@ -33,16 +33,18 @@ interface AsyncTestCluster {
   dependencies: Map<string, string[]>;
 }
 
-const accountDir = '.starknet_accounts_devnet';
-fs.mkdirSync(accountDir, { recursive: true });
-fs.writeFileSync(path.join(BASE_PATH, accountDir, 'starknet_open_zeppelin_accounts.json'), '{}');
-const address = execSync(`${starknetCliCall('new_account', '')} | awk 'NR==1 {printf $3}'`);
+fs.writeFileSync(path.join(BASE_PATH, 'tests', 'cli', 'starknet_open_zeppelin_accounts.json'), '{}');
+const new_account = `${starknetCliCall('new_account', '')} | awk 'NR==1 {printf $3}'`;
+console.log(new_account);
```
Comment (temyurchenko) [inline tests/behaviour/behaviour.test.ts:38]: Do we want that? If we show it for user information, do we want to add some text explaining what it is?

**Review by temyurchenko — APPROVED** [review]

File: tests/behaviour/behaviour.test.ts (hunk 1 above)
Comment (cicr99) [inline tests/behaviour/behaviour.test.ts:38]: Is this necessary? I think you can remove it

File: tests/behaviour/behaviour.test.ts [hunk 2]
```diff
@@ -33,16 +33,18 @@ interface AsyncTestCluster {
...
+const address = execSync(new_account);
 console.log(`New address: ${address}`);
-const addFunds = `curl ${DEVNET_URL}/mint -H "Content-Type: application/json" -d '{"address": "${address}", "amount": 1000000000000000000}'`;
+const addFunds = `curl ${DEVNET_URL}/mint -H "Content-Type: application/json" -d '{"address": "${address}", "amount": 1000000000000000000000000}'`;
 console.log(addFunds);
 console.log(execSync(addFunds).toString());
 console.log('Deploying account:');
-const account_deployed = execSync(starknetCliCall('deploy_account', ''));
+const deployAcc = starknetCliCall('deploy_account', '')
+console.log(deployAcc)
```
Comment (cicr99) [inline tests/behaviour/behaviour.test.ts:46]: Same with this one

File: tests/behaviour/behaviour.test.ts [hunk 3]
```diff
@@ -33,16 +33,18 @@ interface AsyncTestCluster {
   dependencies: Map<string, string[]>;
 }
 
-const accountDir = '.starknet_accounts_devnet';
-fs.mkdirSync(accountDir, { recursive: true });
```
Comment (cicr99) [inline tests/behaviour/behaviour.test.ts:37]: Is this not needed anymore?

**Comment by piwonskp** [comment]
@cicr99 @murcake I've added `console.log` because I constantly keep getting errors on those steps so it's useful for debugging. Let's keep it for a while, since the solution is hackish it is better to be verbose

File: tests/behaviour/behaviour.test.ts (hunk 3 above)
Comment (piwonskp) [reply tests/behaviour/behaviour.test.ts:37]: Yes, the location is `tests/cli` so until someone deletes the directory from repository it's not needed

**Comment by temyurchenko** [comment]
> @cicr99 @murcake I've added `console.log` because I constantly keep getting errors on those steps so it's useful for debugging. Let's keep it for a while, since the solution is hackish it is better to be verbose

I'm suggesting to add some context to the debug output, as opposed to just a raw number, or whatever they are. I imagine it would help in debugging or at least reduce initial confusion.

**Comment by piwonskp** [comment]
> > @cicr99 @murcake I've added `console.log` because I constantly keep getting errors on those steps so it's useful for debugging. Let's keep it for a while, since the solution is hackish it is better to be verbose
> 
> I'm suggesting to add some context to the debug output, as opposed to just a raw number, or whatever they are. I imagine it would help in debugging or at least reduce initial confusion.

Actually the output is just a command so you know what command was run thus know which has failed

**Comment by temyurchenko** [comment]
> > > @cicr99 @murcake I've added `console.log` because I constantly keep getting errors on those steps so it's useful for debugging. Let's keep it for a while, since the solution is hackish it is better to be verbose
> > 
> > 
> > I'm suggesting to add some context to the debug output, as opposed to just a raw number, or whatever they are. I imagine it would help in debugging or at least reduce initial confusion.
> 
> Actually the output is just a command so you know what command was run thus know which has failed

Oh yeah, I parsed it as actual bash evaluation accidentally

**Review by rodrodros — APPROVED** [review]

**Pull request merged**